watchfiles/filters.py

```diff
@@ -82,6 +82,8 @@ class DefaultFilter(BaseFilter):
         'site-packages',
         '.idea',
         'node_modules',
+        '.mypy_cache',
+        '.pytest_cache',
```

Comment:
```suggestion
        '.pytest_cache',
        '.hypothesis',
```

Might as well add `.hypothesis` as well.

Reply:
also:
* `node_modules`
* `.DS_Store`

???

But probably the long term solution is to read `.gitignore`.

Reply:
`.DS_Store` files are common in MacOS.
`node_modules` are common for projects that involves doing cloud deployment.
No harms for keeping those afaic.

Reply:
I'm not sure I understand your logic here - why should `node_modules` not be included here?

Also, `.DS_Store` changes should definitely be ignored, but it should go in `ignore_entity_patterns` not this tuple.

Reply:
It seems like I misunderstood your "???". `node_modules` is already in the list, so I thought your question was why it was in the list, not that you wanted to add to it.

Reply:
Oh, I'm so sorry 🤦.

Somehow I missed them, my mistake.